scripts/special-issues.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This script is designed to be called from 'script-mailer.sh'
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+HOST=prod--xpub--1.elifesciences.org
+
+SEARCH_TERM=neuroblast
```

Comment:
Would it be useful to make this an arg for future special issues?

Reply:
+1